zekrom-vale/ZekDex
# Documentaion and catch rate : Main to stable pull

## Pull Request for ZekDex

**Issue Being Addressed**
None

**Summary of Changes**
This pull request includes several updates to the ZekDex repository. On May 8, 2024, a description and maintainer were added, issue templates were updated, a pull request template was added, and a CONTRIBUTING.md file was created. On May 9, 2024, catch rate data was added to the repository.

**Update Data or @importFrom Directives**
The following `@importFrom` directives were added:
- `importFrom(dplyr,any_of)`
- `importFrom(dplyr,as_tibble)`
- `importFrom(dplyr,coalesce)`
- `importFrom(purrr,walk)`
- `importFrom(tidyr,separate)`

The catch rate data can be updated in the future by scraping the latest data from the source and running the data cleaning and formatting scripts.

**Testing Instructions**
To test these changes, clone the repository and check the functionality of the package. Ensure that the catch rate data is correctly incorporated and that the new `@importFrom` directives are working as expected.

**Additional Notes**
This pull request includes significant updates to the ZekDex repository, including the addition of new data and several enhancements to the repository's documentation and templates. These changes are expected to improve the usability and functionality of the package.
